cicd/unit_test_example_no_django.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
```

Comment:
`unit_test.sh` needs to `cd $APP_ROOT` at the start and `cd -` at the end.

This is so the `source` lines in pr_check.sh that come after `source unit_test.sh` continue to work.